src/vue/components/word-list/word-language-panel.vue

```diff
@@ -58,6 +58,12 @@
                 {{ l10n.getText('WORDLIST_DOWNLOAD_BUTTON') }}
               </button>
             </alph-tooltip>
+            <div class="alpheios-wordlist-download-with-filters alpheios-checkbox-block" data-alpheios-ignore="all">
+              <input id="alpheios-wordlist-download-with-filters-input" type="checkbox" v-model="downloadWithFilter">
+              <label for="alpheios-wordlist-download-with-filters-input">
+                with filtering
```

Comment:
please move text to messages file

Reply:
Done - I have missed it myself, thank you!